data/OIG-chip2/prepare.sh

```diff
@@ -0,0 +1 @@
+wget https://huggingface.co/datasets/laion/OIG/resolve/main/unified_chip2.jsonl -O data/OIG-chip2/unified_chip2.jsonl
```

Comment:
Nice. Super simple. Please update the output directory to be relative to the script. Here's one way to do it.

```suggestion
DIR=$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)

wget https://huggingface.co/datasets/laion/OIG/resolve/main/unified_chip2.jsonl -O ${DIR}/data/OIG-chip2/unified_chip2.jsonl
```

Reply:
Updated with ${DIR}/data...
Shouldn't BASH_SOURCE[0] gives the path of the .sh file, and then we'd save the data file as ${DIR}/unified_chip2.json?